catalog/task/update-infra-deployments/0.1/README.md

```diff
@@ -0,0 +1,20 @@
+# update-infra-deployments
+
+* This task clones redhat-appstudio/infra-deployments GH repository.
+* It then runs a script obtained from the infra-deployment-update-script key in extraData which can modify text files.
+* It then generates pull-request for redhat-appstudio/infra-deployments repository using the modified files.
+
+
+## Parameters
+| Name                    | Description                                                                                  | Required | Default Value                                                                                                                                    |
+|-------------------------|----------------------------------------------------------------------------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| extraDataJsonPath       | Path to extraData json file. It should contain a key called 'infra-deployment-update-script' | false    | $(workspaces.data.path)/extra_data.json                                                                                                          |
+| snapshotPath            | Path to snapshot json file                                                                   | false    | $(workspaces.data.path)/mapped_snapshot.json                                                                                                     |
+| originRepo              | URL of github repository which was built by the Pipeline                                     | true     |                                                                                                                                                  |
+| revision                | Git reference which was built by the Pipeline                                                | true     |                                                                                                                                                  |
+| targetGHRepo            | GitHub repository of the infra-deployments code                                              | false    | redhat-appstudio/infra-deployments                                                                                                               |
```

Comment:
If this is a parameter (which I think is great), can we update the task name? Maybe like `update-github-repo` or `create-github-pr` or something?

Reply:
We could always rename at a later date anyways I suppose

Reply:
yes we can